README.md

```diff
@@ -108,7 +108,7 @@ A more detailed list of available CLI arguments for the tool can be found [here]
 
 With an SBOM file in hand, use the tool to validate the output file with the command:
 
-```
+```shell
```

Comment:
In this case, we want the < and > to be placeholders, not literals, so we want the generic formatting